addons-cluster/zookeeper/templates/cluster.yaml

```diff
@@ -14,20 +14,26 @@ spec:
       serviceAccountName: {{ include "kblib.serviceAccountName" . }}
       {{- include "kblib.componentMonitor" . | indent 6 }}
       {{- include "kblib.componentResources" . | indent 6 }}
+      {{- include "kblib.componentStorages" . | indent 6 }}
+      {{- include "kblib.componentServices" . | indent 6 }}
       volumeClaimTemplates:
-        - name: data
+        - name: data # ref clusterDefinition components.containers.volumeMounts.name
```

Comment:
"kblib.componentStorages" includes such an entry: 
```
volumeClaimTemplates
  - name: data
```
Why there is a duplicate one?

Reply:
there is a missing commit, so I pushed it